governance.md

```diff
@@ -51,10 +51,26 @@ Subcommittees may be established by the Steering Council to assist in managing t
 
 ### Editorial policies and processes
 
-Individual journals are responsible for their own editorial process, including their editorial board composition. However, by being an Open Journals publication, each journal must:
+Individual journals are responsible for their own editorial process, including their editorial board composition. However, by being an Open Journals publication, each journal:
+
+**Must**
+
+- Be open access ([Diamond/Platinum or Gold](https://en.wikipedia.org/wiki/Open_access#Definitions)).
+- License journal content (e.g., articles) under a [CC-BY license](https://creativecommons.org/licenses/by/4.0/). 
+- Follow an open peer review process.
+- Publish a cost model (e.g., https://joss.theoj.org/about#costs).
+- Have a code of conduct (e.g., [Contributor Covenant](https://github.com/openjournals/joss/blob/master/CODE_OF_CONDUCT.md)) and a documented process for dealing with infringements. 
```

Comment:
The JOSS code of conduct refers to "project maintainers". I guess it was written for software development projects, not journals. In the context of JOSS, it is misleading because it could refer to the maintainers of the project that is the topic of a submitted article. Maybe replace by "editorial board",  or some subset of the editorial board?

Reply:
@khinsen - Is your concern with this document (governance.md) or a different document?

Reply:
My direct concern is with this document. It recommends a code of conduct for future journals that I consider badly worded. As someone involved with a candidate journal, that matters for me. It may well be that it matters for the JOSS editorial board as well, but that's not for me to decide.

In other words: if JOSS decides to update its code of conduct along these lines, I am fine with recommending it as a model for others. As it is, I think it's not a good model.

Reply:
Thanks, I now understand your concern. Do you want to suggest changes in the JOSS CoC that would address your issues?  Or suggest a different CoC we should mention as an example in this document?

Reply:
How about citing the [Contributor Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/) Web site directly? Since version 2, it says "community leaders" instead of "project maintainers", which looks a lot better.

Reply:
That's fine with me

Reply:
If you have suggestions you want to make for https://github.com/openjournals/joss/blob/master/CODE_OF_CONDUCT.md as well, please do